library/src/window_impl.cxx

```diff
@@ -184,24 +181,12 @@ window_impl::window_impl(const options& options, const std::optional<Type>& type
   }
   else if (type == Type::EXTERNAL)
   {
-#if F3D_MODULE_EXTERNAL_RENDERING
-    vtkNew<vtkExternalOpenGLRenderWindow> extWin;
-    extWin->AutomaticWindowPositionAndResizeOff();
-    this->Internals->RenWin = extWin;
-#else
-    throw engine::no_window_exception(
-      "Window type is external but F3D_MODULE_EXTERNAL_RENDERING is not enabled");
-#endif
+    this->Internals->RenWin = vtkSmartPointer<vtkF3DExternalRenderWindow>::New();
   }
   else if (type == Type::EGL)
   {
 #if defined(VTK_OPENGL_HAS_EGL) && VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 3, 20240914)
     this->Internals->RenWin = vtkSmartPointer<vtkEGLRenderWindow>::New();
-#ifdef __ANDROID__
-    // Since F3D_MODULE_EXTERNAL_RENDERING is not supported on Android yet, we need to call
-    // this workaround. It makes vtkEGLRenderWindow external if WindowInfo is not nullptr.
```

Comment:
does that work on android with your window ?

Reply:
I cannot test it right now unfortunately because my Android setup is broken.  
I do plan to make an Android pass before our 3.0 release.

Reply:
Sounds fine to me as we consider the java/android stuff to be experimental